README.md

```diff
@@ -787,6 +787,9 @@ A: First, you'll need to get a public/private key pair to serve over SSL.
 These need to be in a directory accessible by Atlantis. Then start `atlantis server` with the `--ssl-cert-file` and `--ssl-key-file` flags.
 See `atlantis server --help` for more information.
 
+**Q: How can I get Atlantis up and running on AWS?**
+
+A: There is [terraform-aws-atlantis](https://github.com/terraform-aws-modules/terraform-aws-atlantis) project where complete Terraform configurations for running Atlantis on AWS Fargate are hosted. Tested, maintained, and free.
```

Comment:
Can this say "Tested and maintained" because I'm worried people might think the hosting is free which (although cheap) it is not!